### ISSUE TITLE
Door timing can cause player to be trapped

### ISSUE THREAD

**Issue by Moe-Ronickah** [Wynncraft/Issues#1853]
I cannot put the coordinates here, so, reading the mob identifiers (Tempest Lv. 27) will be all that anyone gets to recognize the place.  If that is not enough, any builder or mod can contact me through email or a forum conversation.  I opened the door to this place, walked for the door, and got caught in the redstone circuit and sticky pistons.  Teleport scrolls don't work here, so /kill was my escape route.  I believe that the door close delay should be longer, or maybe a sign that says "Run for it!"

![2015-04-29_03 38 14](https://cloud.githubusercontent.com/assets/11842380/7387610/357ae026-ee24-11e4-9129-790fa38be15e.png)

**Comment by ThunderRemix** [comment]
I believe this is the "Kraken Island" located at x -254, z -2678

**Comment by Moe-Ronickah** [comment]
Yes, it is.

**Comment by Moe-Ronickah** [comment]
TR, what does "If that is not enough, any builder or mod can contact me through email or a forum conversation." mean to you?  It means "don't publish the coordinates"!  Oh, @Tama63 , would you remove TR's coordinates, please?  Thanks, in advance.

:)

**Comment by ThunderRemix** [comment]
@Moe-Ronickah 

Alright, I've decided to remove them myself due to your complaining. I find it rather rude that you ask someone else to do remove something when I'm perfectly capable of doing it myself. 

As far as the coordinates go, the island is even on the wynncraft map and it's location along with the mobs it contains has already been publicly disclosed on the forums.

"I cannot put the coordinates here, so, reading the mob identifiers (Tempest Lv. 27) will be all that anyone gets to recognize the place." (I took this as "I forgot to get the coordinates to the general area or the exact place, so, reading the mob identifiers (Tempest Lv. 27) will be all that anyone gets to recognize the place." )

Thanks for reading! :)

**Comment by Moe-Ronickah** [comment]
TR, I am aware of all that you said, however, you missed my point, thus; I cannot put the coordinates up, nor can I put the exact location up, because to do so, could easily get oneself banned, by the Ban Hammer of Greatness himself, Salted.  Stop being overly sensitive.  I know that you were just trying to help, however, I believe that my statement  "If that is not enough, any builder or mod can contact me through email or a forum conversation" makes it quite clear that I know exactly where I was.  Now, what sites are we not allowed to publish the location of, upon pain of being banned?  Don't write it out in your next response, OK?  Why not?  Because you have named the place and that is also bannable.  So, if you would be so kind as to remove the name of the place, first, then, if you wish, state the type of site, that would not be a problem.  Maybe I'm wrong, however, I don't believe so, therefore, I'm staying out of trouble, and I'm trying to keep you out of trouble, too.

Now, as to why I asked "someone else" (LOL) to remove the coordinates, how am I supposed to know when you next will check in on the thread?  Remember, I'm trying to keep myself out of trouble, and you, too.

**Comment by ThunderRemix** [comment]
"Because you have named the place and that is also bannable."
...LOL LOL LOL LOL LOL LOL lol lol ...lo...l

Fine. I know what site is may or may not be. it wasn't suggested until you mentioned it.

**Comment by Moe-Ronickah** [comment]
Thank you, my young padawan.  Now you know why I was semi-cryptic when I posted it.  <grin>

**Comment by epicman739** [comment]
@Moe-Ronickah Wait you can get banned for naming that location? Why? Is there a new rule that I didn't hear about? Or is there a relic there?
So many questions...

**Comment by ThunderRemix** [comment]
I don't think you can get banned for just giving it a name, but whatever.

**Comment by Moe-Ronickah** [comment]
TR and e739, quoting from Salted initial post concerning relics"

"And finally, one last change. We added a new rule for the forum and wiki: It is not allowed to post the coordinates of the relics, or a screenshot of them. You are allowed to talk about them, what you've found and their general locations, but not their exact coordinates or how to find them. We want you guys to discover their secrets without using guides or a wiki."

Ergo, I stand corrected, and I apologize.  It *is* OK to talk about their general locations.  *Walks off, SMH.*

**Comment by Tama63** [comment]
Best speak to a builder about this /cc @AckroWynn (same chat?)

**Comment by AckroWynn** [comment]
There are things we, as builders, can fix and there are things we can't.
I could go and see if we can make the redstone signal longer, but we can never really prevent you from getting into the redstone. If you stand in the door when it closes, there is always a chance you will get caught in the redstone.

Thanks for reporting the issue, however!

**Comment by Moe-Ronickah** [comment]
My pleasure, re the thanks.

I understand that not everything in MC is doable; I've trapped myself in enough doors and redstone circuits in my own builds and experiments.  The real problem is that I walked to the door, however, I didn't have enough time to get through the door.  As I first suggested, perhaps increasing the redstone signal, or placing a sign that says "Hurry!" would help future players.

I'm closing this issue, as I believe it has gotten to the right people.

Thanks, everyone.
